charts/k6-operator/values.yaml

```
@@ -116,13 +116,84 @@ namespace:
 # required: false
 # type: object
 # @schema
-prometheus:
+metrics:
   # @schema
   # required: false
-  # type: boolean
+  # type: object
+  # description: related to ServiceMonitor
   # @schema
-  # prometheus.enabled -- enables the prometheus metrics scraping (default: false)
-  enabled: false
+  serviceMonitor:
+    # @schema
+    # required: false
```

```suggestion
    # required: true
```
Maybe this is a nit, but if one adds `metrics.serviceMonitor`, they probably should also set `enabled`, otherwise there won't be any point to it.